requirements.txt

```diff
@@ -5,3 +5,4 @@ pytest-cov
 sympy
 numpy
 find_libpython
+scikit-build
```

Comment:
looks good - some scripts in the repo explicitly install skbuild and then requirements.txt. We can remove the first one.

Reply:
done